app/components/patients/Patient.js

```diff
@@ -0,0 +1,138 @@
+// import redux, react
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Helmet from 'react-helmet';
+import { Link, browserHistory } from 'react-router';
+
+//Material UI
+import { FontIcon, Table, TableHeader, TableHeaderColumn, TableBody, TableRow, TableRowColumn } from 'material-ui';
+import { StepsRaisedButton, StepsActionButton } from '../material-style';
+import { toolbar, textLight, active, errorText } from '../colors';
+import moment from 'moment';
+
+// custom components
+import ProgressGraph from '../dashboard/ProgressGraph'
+import SidePanel from '../widgets/SidePanel';
+import InfoItem from '../widgets/InfoItem';
+import ConfirmDialog from '../widgets/ConfirmDialog';
+
+// redux and utils
+import { deletePatient } from '../../reducers/patients'
+import { fullName } from '../../utils';
+
+// table style
+const colHeaderStyle = { color: textLight, textAlign: "center", fontWeight: "bold", fontSize: "14px" };
+
+export class Patient extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      isOpen: false
+    }
+    this.dialogToggle = this.dialogToggle.bind(this);
+  }
+
+  dialogToggle(){
+    this.setState({isOpen: !this.state.isOpen});
+  }
+
+  render() {
+    const { currentPatient } = this.props;
+    if (!Object.keys(currentPatient).length) return null;
+
+    const patientName = fullName(currentPatient);
+    const sortedPlans = currentPatient.plans.sort((a, b) => new Date(b.created_at) - new Date(a.created_at));
+    console.log(sortedPlans)
+    const buttons = [
+      <StepsRaisedButton
+      label="Chat"
+      fullWidth={true}
+      onTouchTap={ () => browserHistory.push('/messages') }
+      onClick={() => console.log('route to chat')} />,
```

Comment:
Why is there a onTouchTap and an OnClick ? Think you meant to remove the OnClick?